android/src/main/java/com/reactcommunity/rndatetimepicker/RNDate.java

```diff
@@ -1,13 +1,19 @@
 package com.reactcommunity.rndatetimepicker;
 
 import java.util.Calendar;
+import java.util.TimeZone;
 import android.os.Bundle;
 
 public class RNDate {
   private Calendar now;
 
   public RNDate(Bundle args) {
     now = Calendar.getInstance();
+    if (args != null && args.containsKey("timeZoneOffsetInMinutes")) {
```

Comment:
can you please extract "timeZoneOffsetInMinutes" into a constant, same as `RNConstants.ARG_VALUE`?